operations/restrictions.md

```diff
@@ -439,42 +439,6 @@ The rules that prevent the restriction from applying to a reservation, even when
 | `MinPrice` | [Currency value (ver 2018-06-07)](_objects.md#currency-value-ver-2018-06-07) | optional | Value of the minimum price per time unit. |
 | `MaxPrice` | [Currency value (ver 2018-06-07)](_objects.md#currency-value-ver-2018-06-07) | optional | Value of the maximum price per time unit. |
 
-## ~~Delete restrictions~~
-
-> ### Deprecated!
-> This operation is [deprecated](../deprecations/README.md). Please use [Clear restrictions](restrictions.md#clear-restrictions) instead.
-
-Removes restrictions from the service. This operation is intended to be used alongside [Add restrictions](restrictions.md#set-restrictions).
-
-### Request
-
-`[PlatformAddress]/api/connector/v1/restrictions/delete`
-
-```javascript
-{
-  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
-  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
-  "Client": "Sample Client 1.0.0",
-  "RestrictionIds": [
-    "af4949ce-c061-4f27-89f9-5a6a9ef725a7",
-    "e2f8aa29-0c09-4097-801c-7888c9fb57ae"
-  ]
-}
-```
-
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `ClientToken` | string | required | Token identifying the client application. |
-| `AccessToken` | string | required | Access token of the client application. |
-| `Client` | string | required | Name and version of the client application. |
-| `RestrictionIds` | array of string | required | Unique identifiers of the `Restriction`. |
-
-### Response
-
-```javascript
-{}
-```
-
 ## Set restrictions
```

Comment:
Two issues here:

1. The newlines between paragraphs in the description have disappeared, this is causing problems with the Markdown.
2. The **Restricted!** breakout at the top should be removed, because this is now the official way to add restrictions.